src/app/components/multiple-card-overview/multiple-card-overview.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="mat-dialog">
+  <div class="dialog-header">
+    <h2 mat-dialog-title>{{ title1 }} VS {{ title2 }}</h2>
+    <mat-dialog-actions>
+      <button mat-icon-button color="warn" (click)="onClose()">
+        <mat-icon>close</mat-icon>
+      </button>
+    </mat-dialog-actions>
+  </div>
+
+  <mat-dialog-content>
+    <div class="dialog-body">
+      <!-- imgs -->
+      <div class="body-img">
+        <mat-grid-list cols="2" rowHeight="1:1">
+          <mat-grid-tile [rowspan]="1">
+            <img
+              mat-card-image
+              class="body-img-item"
+              src="{{ pokemonImage1 }}"
+            />
+          </mat-grid-tile>
+          <mat-grid-tile [rowspan]="1">
+            <img
+              mat-card-image
+              class="body-img-item"
+              src="{{ pokemonImage2 }}"
```

Comment:
use inputs instead of interpolation, if no input is available for the attribute use [attr.atributename]